executor/executor_test.go

```diff
@@ -6481,3 +6481,17 @@ from ssci right join csci on (ssci.customer_sk=csci.customer_sk
                                and ssci.item_sk = csci.item_sk)
 limit 100;`)
 }
+
+func TestIssue42298(t *testing.T) {
```

Comment:
It should be put into ```executor/issuetest``` or append the existed test cases. ```executor``` has too many test cases.

Reply:
Done. PTAL